src/App.js

```diff
@@ -1,18 +1,80 @@
 import React, {useState} from "react";
-import Form from "./components/Form";
 import {nanoid} from "nanoid";
+import Form from "./components/Form";
+import Todo from "./components/Todo";
+
+const FILTER_MAP = {
+  All: () => true,
+  Active: task => !task.completed,
+  Completed: task => task.completed
+}
+
+const FILTER_NAME = Object.keys(FILTER_MAP)
 
 function App(props) {
-  const [task, setTasks] = useState(props.tasks);
+  const [tasks, setTasks] = useState(props.tasks);
+  const [filter, setFilter] = useState('All');
```

Comment:
`setFilter` is not used.

Reply:
Filtering of the task is a separate user story.